### ISSUE TITLE
[Sequester] Feature request: Add a magic label to remove any linked work item

### ISSUE THREAD

**Issue by BillWagner** [dotnet/docs-tools#416]
Add a new label, like `unQuest` that does the following:

If an issue is imported into DevOps, change the work item's state to "removed", and un-link the issue.

---
[Associated WorkItem - 360309](https://dev.azure.com/msft-skilling/Content/_workitems/edit/360309)

**Comment by BillWagner** [comment]
As I'm working on this, I think that the issue shouldn't be un-linked. We're not closing the issue, and if it every got scheduled, the link would mean the code could just change the status, set a new iteration, etc. That would already just work.

@cmastr What do you think?

**Comment by cmastr** [comment]
Yes I agree with you on that, keeping the link is better. That way we still have an electronic paper trail even when we decide not to do something (right now or ever).
